src/utils.js

```diff
@@ -57,7 +61,14 @@ export function isPointType(value) {
 export function setPixiValue(instance, propName, value) {
   if (isPointType(instance[propName]) && isPointType(value)) {
     // Just copy the data if a Point type is being assigned to a Point type
-    instance[propName].copy(value);
+
+    // Copy was deprecated and renamed to copyForm in PIXI 5.0
+    // Use whichever exists
+    if (hasCopyFunction(instance[propName])) {
```

Comment:
I think it would be better to extract this logic to a function called for example `copyPoint(instance, propName, value)` than to just create and use `hasCopyFunction` function.

Reply:
I've made the changes.

Reply:
Splendid, but you forgot about the test you wrote ;) I'll merge and release this as soon as the build is passing.

Reply:
whoops

Reply:
I've updated the tests and included a check for whether it's using PIXI v4 or v5.